src/components/Header/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react"
+import { Link } from "react-router-dom"
+import { IoMdCart } from "react-icons/io"
+import Sidebar from "./Sidebar"
+import { GiHamburgerMenu } from "react-icons/gi"
+import styles from "./Header.module.scss"
+
+const Header = () => {
+  const [open, setOpen] = useState(false)
+
+  return (
+    <header className={styles.header}>
+      <nav className={styles.main_nav}>
+        <GiHamburgerMenu
+          className={styles.menubar}
+          onClick={() => setOpen(true)}
+        />
+        <img
+          src="../../../static/images/logo_2x.png"
+          alt="SABKA_BAZAAR"
+          className={styles.logo}
+        />
+        <nav className={styles.navigations_one}>
+          <Link to="/">Home</Link>
+          <Link to="/products">Products</Link>
+        </nav>
+      </nav>
+      <div className={styles.cart_menu}>
+        <nav className={styles.navigations_two}>
```

Comment:
ideally a page should have just one nav, all other navigation items could be ul/ol

http://html5doctor.com/nav-element/

The nav element represents a section of a page that links to other pages or to parts within the page: a section with navigation links. Not all groups of links on a page need to be in a nav element only sections that consist of major navigation blocks are appropriate for the nav element. In particular, it is common for footers to have a list of links to various key parts of a site, but the footer element is more appropriate in such cases, and no nav element is necessary for those links.

Reply:
all the Link tags moved inside nav tag.